client/src/pages/signin.js

```diff
@@ -88,7 +93,7 @@ export default function Signin() {
     }
 
     if (redirect) {
-        return <Redirect to='/dashboard' />
+        return <Redirect to='/' />
```

Comment:
This redirect didn't happen for me, I think it's because this component is not mounted when the condition is true. Changing the approach in App.js should resolve this as well. You can probably combine this variable with authorized since they would always be equal

Reply:
Changed to Authorized. Thank you.